src/plugins/share/public/services/short_url_redirect_app.ts

```diff
@@ -8,24 +8,43 @@
 
 import { CoreSetup } from 'kibana/public';
 import { getUrlIdFromGotoRoute, getUrlPath, GOTO_PREFIX } from '../../common/short_url_routes';
+import {
```

Comment:
why were the tests for this file deleted ?

Reply:
There was one test case for this file, but it was not very useful it mostly tested the mocks of the test and those mocks became invalid after refactor. I don't believe we need that test.